### DISCUSSION TITLE
If u have inspect element, this is a nice troll to extensions

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1237] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
`setInterval(function() {
    chrome.management.get('replace with extension id', function(extension) {
        if (extension) {
            const isEnabled = extension.enabled;
            chrome.management.setEnabled('replace with extension id', !isEnabled);
        }
    });
}, 145);
`
it should be run on chrome://extensions

**Comment by buddyted** [comment]
how do I inspect element

**Reply by schoolexploitkid** [reply]
its mostly blocked but u can troll extensions on personal computers

**Comment by SlitheringLizard1** [comment]
what does this even do

**Reply by schoolexploitkid** [reply]
turn ext on and off continuously

**Reply by jazzymakesstuff** [reply]
why would anyone ever do this 😭

**Reply by SlitheringLizard1** [reply]
it could crash an extension for all students or possibly crash the teacher's thing

**Comment by akabutnicer** [comment]
wait til bro finds out `chrome.permissions.contains({permissions:{length:Infinity}})`...

**Reply by SlitheringLizard1** [reply]
oh no that is so broken

**Reply by akabutnicer** [reply]
forgot api for a minute

**Reply by SlitheringLizard1** [reply]
I almost don't want to try your code

**Reply by akabutnicer** [reply]
just hangs the extension (which, if missing the web request API, can unrestrict websites)

**Reply by akabutnicer** [reply]
honestly even smaller, `for(;;)1` iirc

**Reply by thatboyo** [reply]
how do i use your code?
